### ISSUE TITLE
doors

### ISSUE THREAD

**Issue by urgedpanda** [JMaverick16/Airbus_A330#4]
cargo doors appear on both sides of the plane

**Comment by Marsdolphin** [comment]
It should be a livery problem - the white one was wrong so all the liveries made using that template are all wrong :(

**Comment by Marsdolphin** [comment]
So, because of that, we need to redo almost every livery :(
Which is a _**TREMENDOUS**_ amount of work

**Comment by JMaverick16** [comment]
No, you don't. A layer can be added worh Gimp. Probably won't gonna happen.

**Comment by Marsdolphin** [comment]
Ok. idk about GIMP

**Comment by Marsdolphin** [comment]
Fixing the paintkits

**Comment by Octal450** [comment]
No fix for the time being.
